docroot/modules/custom/foia_webform/src/AgencyLookupServiceInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+/**
+ * Interface AgencyLookupServiceInterface.
```

Comment:
This should be some sort of minimal description about the, rather than stating that it's an interface. Something like: `Provides an interface defining an agency lookup service.` You can check out modules in core for other examples.